app/Http/Kernel.php

```diff
@@ -53,6 +53,7 @@ class Kernel extends HttpKernel
      * @var array<string, class-string|string>
      */
     protected $routeMiddleware = [
+        'dbcreate' => \App\Http\Middleware\DBCreate::class,
```

Comment:
Small thing, but I would think this should be `dbCreate` instead. Not a huge deal though